README.md

```diff
@@ -151,3 +151,31 @@ To test the NGINX Kubernetes Gateway run:
 ```
 make unit-test
 ```
+
+# Release Process for NGINX Kubernetes Gateway
+
+### NGINX Kubernetes Gateway uses semantic versioning for its releases. For more information see https://semver.org.
+
+Warning: Major version zero (0.y.z) is reserved for development, anything MAY change at any time. The public API is not stable."
```

Comment:
Why is there a quote at the end of the sentence here?

Reply:
That shouldn't have been there.  I removed it.

Reply:
It shouldn't be there.  I removed it.

Reply:
It shouldn't be there.  I deleted it.